src/pages/characters/[[...params]].js

```diff
@@ -0,0 +1,77 @@
+import { makeStyles } from "@material-ui/core/styles"
+import { Typography } from "@material-ui/core"
+import { useSelector } from "react-redux"
+import { useRouter } from "next/router"
+import { find } from "lodash"
+
+import CustomTable from "@/components/CustomTable"
+import AddNewButton from "@/components/AddNewButton"
+import RightSideDrawer from "@/components/RightSideDrawer"
+import CharacterForm from "@/components/CharacterForm"
+import { characters } from "@/selectors/characterSelectors"
+import { formEnum } from "@/enums/formEnum"
+
+const useStyles = makeStyles((theme) => ({
+  body: {
+    display: "grid",
+    placeItems: "center",
+    gap: "10px",
+    padding: "25px",
+  },
+  button: {
+    placeSelf: "end",
+  },
+}))
+
+export default function Home() {
+  const classes = useStyles()
+
+  const data = useSelector(characters)
+
+  const router = useRouter()
+  const { params } = router.query
+
+  const columns = [
+    { header: "Name", field: "name" },
+    { header: "Eye Color", field: "eyeColor" },
+    { header: "Height", field: "height" },
+    { header: "Gender", field: "gender" },
+    { header: "Birth Year", field: "birthYear" },
+    { header: "Homeworld", field: "homeworld.name" },
+    { header: "Species", field: "species.name" },
+    {
+      header: "Number of Films",
+      field: "filmConnection.totalCount",
+    },
+  ]
+
+  let isOpen = false
+  if (params) isOpen = !!find(formEnum, (value) => value.includes(params[0]))
```

Comment:
You should never be doing things like this. We very rarely ever need the `let` keyword and this is not a good way to determine if the form is open or not. Consider using the `useMemo` hook instead